config/tips.txt

```diff
@@ -31,56 +34,63 @@ As a Medical Doctor, you must target the correct limb and be on help intent when
-As a Geneticist, check the door to your workspace once in a while to see if there's anyone to clone.
-As a Geneticist, becoming a hulk makes you capable of dealing high melee damage, stunlocking people, and punching through walls. However, you can't fire guns, will lose your hulk status if you take too much damage, and are not considered a human by the AI while you are a hulk.
+As a Geneticist, check the door to your workspace periodically to see if there's anyone to clone.
+As a Geneticist, becoming a hulk makes you capable of dealing high melee damage, stunlocking people, and punching through walls. However, you can't fire guns, will lose your hulk status if you take too much damage, and are not considered a human by the AI as a hulk.
 As the Virologist, your viruses can range from healing powers so great that you can solo a blob, or diseases so dangerous they can kill the entire crew with airborne spontaneous combustion. Experiment!
-As the Research Director, you can repair and even revive AIs by loading them into an intelicard, and then from there into an AI system integrity restorer computer.
+As the Virologist, you only require small amounts of vaccine to heal a sick patient. Work with the Chemist to distribute your cures more efficiently.
+As the Research Director, you can repair and even revive AIs by loading them into an intellicard, and then from there into an AI system integrity restorer computer.
 As the Research Director, you can lock down cyborgs instead of blowing them up. Then you can have their laws reset or if that doesn't work, safely dismantled.
-As a Scientist, you can inject yourself with the mutation toxin extracted from green slimes to become a slime man, who will never be attacked by slimes.
+As a Scientist, you can inject yourself with the mutation toxin extracted from green slimes to become one of many races, including a slimeperson, who will never be attacked by slimes and can split!
 As a Scientist, you can maximize the number of uses you get out of a slime by feeding it slime steroid, created from purple slimes, while alive. You can then apply extract enhancer, created from cerulean slimes, on each extract. It is possible to get 9 times as much out of a slime this way.
 As a Scientist, you can disable anomalies by scanning them with an analyzer, then send a signal on the frequency it gives you with a remote signalling device. This will leave behind an anomaly core, which is good for research or the construction of the Phazon mech!
 As a Scientist, researchable machine parts can seriously improve the efficiency and speed of machines around the station. In some cases, it can even unlock new functions.
+As a Scientist, with high enough bluespace research, you can build machines that can teleport objects and people all around the station! Look into setting up a telescience department if you can.
 As a Roboticist, you can not only build cyborgs, but also drones, bots, and even mechs!
```

Comment:
Dorsi is talking about the very bottom line there "As a Roboticist, you can not only build cyborgs, but also drones, bots, and even mechs!" unless they've bucked their trend.